src/pages/GeneratingPage.tsx

```diff
@@ -30,11 +35,21 @@ export const GeneratingPage = () => {
             name="classToGenerate"
             ref={register}
           >
-            {mapperClassList.map((dndClass, index) => (
-              <option key={index} className="text-capitalize" value={dndClass}>
-                {dndClass === "class" ? "random" : dndClass}
-              </option>
-            ))}
+            {classes != null
+                  ? <Fragment>
+                    <option key={"dndClass" + 0} className="text-capitalize" value={'class'}>
+                      {"Random"}
+                    </option>
+                    {classes && classes.map((dndClass: any, index: number) => (
+                      <option key={"dndClass" + (index + 1).toString()} className="text-capitalize" value={dndClass.index}>
```

Comment:
I think just `index` is enough, and you don't have to put `any` and `number` type def there, TS can automatically detect it

Reply:
I had a problem where there were repeated keys. 
So if I had a list of classes, and a list of levels to choose from, the keys would be "bard1" and "level1" instead of "1" and "1".

Reply:
ah got you! Nice catch